mmedit/models/__init__.py

```diff
@@ -6,6 +6,7 @@
                           MultiLayerDiscriminator, PatchDiscriminator)
 from .data_preprocessors import (EditDataPreprocessor, GenDataPreprocessor,
                                  MattorPreprocessor)
+from .diffusers import *  # noqa: F401, F403
```

Comment:
Place `diiffusers` at the same folder level as `editor` may not be a good idea.

Reply:
``diffusers`` will be placed under ``editor``.